tests/test_dcrAssembleCoadd.py

```diff
@@ -0,0 +1,295 @@
+            model.getVariance().getArray()[:, :] = image
+            model.mask.addMaskPlane("CLIPPED")
+            self.dcrModels.append(model)
+        maskVals = np.zeros_like(imageSum)
+        maskVals[imageSum > 5*noiseLevel] = afwImage.Mask.getPlaneBitMask('DETECTED')
+        for model in self.dcrModels:
+            model.getMask().getArray()[:, :] = maskVals
+        self.mask = self.dcrModels[0].getMask()
+
+    def makeDummyWcs(self, rotAngle, visitInfo=None):
+        """! Make a World Coordinate System object for testing.
+
+        @param[in] rotAngle: rotation of the CD matrix, East from North as a lsst.afw.geom.Angle
+        @param[in] visitInfo: lsst.afw.image.VisitInfo for the exposure.
+
+        @return wcs, a lsst.afw.geom.skyWcs.skyWcs.SkyWcs object.
+        """
+        if visitInfo is None:
+            crval = IcrsCoord(Angle(0.), Angle(0.))
+        else:
+            crval = visitInfo.getBoresightRaDec()
+        crpix = afwGeom.Box2D(self.bbox).getCenter()
+        cd_matrix = makeCdMatrix(scale=self.pixelScale, orientation=rotAngle, flipX=True)
+        wcs = makeSkyWcs(crpix=crpix, crval=crval, cdMatrix=cd_matrix)
+        return wcs
+
+    def makeDummyVisitInfo(self, azimuth, elevation):
+        """! Make a self-consistent visitInfo object for testing.
+
+        For simplicity, the simulated observation is assumed to be taken on the local meridian.
+        @param[in] azimuth: azimuth angle of the simulated observation, as a lsst.afw.geom.Angle
+        @param[in] elevation: elevation angle of the simulated observation, as a lsst.afw.geom.Angle
+
+        @return visitInfo: lsst.afw.image.VisitInfo for the exposure.
+        """
+        lsstLat = -30.244639*afwGeom.degrees
+        lsstLon = -70.749417*afwGeom.degrees
+        lsstAlt = 2663.
+        lsstTemperature = 20.*u.Celsius  # in degrees Celcius
+        lsstHumidity = 40.  # in percent
+        lsstPressure = 73892.*u.pascal
+        lsstWeather = Weather(lsstTemperature.value, lsstPressure.value, lsstHumidity)
+        lsstObservatory = Observatory(lsstLon, lsstLat, lsstAlt)
+        airmass = 1.0/np.sin(elevation.asRadians())
+        era = Angle(0.)  # on the meridian
+        ra = lsstLon + np.sin(azimuth.asRadians())*(Angle(np.pi/2.) - elevation)/np.cos(lsstLat.asRadians())
+        dec = lsstLat + np.cos(azimuth.asRadians())*(Angle(np.pi/2.) - elevation)
+        visitInfo = afwImage.VisitInfo(era=era,
+                                       boresightRaDec=IcrsCoord(ra, dec),
+                                       boresightAzAlt=Coord(azimuth, elevation),
+                                       boresightAirmass=airmass,
+                                       boresightRotAngle=Angle(0.),
+                                       observatory=lsstObservatory,
+                                       weather=lsstWeather
+                                       )
+        return visitInfo
+
+    def testDcrShiftCalculation(self):
```

Comment:
This test doesn't need the dummy images created in `setUp`. Should this test be moved to a different `TestCase` or should the image creation be moved into a different method?